### PR TITLE
Added Footer Design In app.js

### PR DESCRIPTION
**Pull request by Priyanshu-official**
Improve Footer Design #470 

New Footer Design Added In app.js File 
Responsive Work Fine for Both Web and Mobile view

### CODE REVIEW

**Comment by Priyanshu-official** [comment]
@smaranjitghose @anushbhatia Pls Review New PR

File: assets/css/style.css [hunk 1]
```diff
@@ -91,3 +91,211 @@ html {
...
+  font-weight: normal;
+  line-height: 2em;
+  border-radius: 1.6875rem;
+  transition: all 0.5s ease;
+  margin-right: 0.25rem;
+  margin-bottom: 0.25rem;
+}
+
+/********
+Rotate Transition in foote Soical Icon
```
Comment (smaranjitghose) [inline assets/css/style.css:142]: Remove the rotate transition. It appears out of place for these icons

**Comment by smaranjitghose** [comment]
Please use [feather icons](https://feathericons.com/) for the footer

**Comment by smaranjitghose** [comment]
WhatsApp Image 2021-01-17 at 5 41 18 PM

This is the color palette we are using for the website. In case you need help with deciding, what to use where DM @anushbhatia

**Comment by smaranjitghose** [comment]
Also, I know this PR is taking some time to get finalized. This is because we expect some great improvements from you😄Thanks for being patient

**Comment by smaranjitghose** [comment]
Please incorporate the changes listed above in another follow up PR

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**